data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```
@@ -36,6 +36,11 @@
     "^corda.ledger.keys.[0-9]+.signature.spec$": {
       "description": "Optional. The ledger public key signature spec. Can be multiple specified by index.",
       "type": "string"
+    },
+    "^ext\\.[a-zA-Z0-9\\.]+$": {
```

Is this supposed to be
```suggestion
    "^ext\.[a-zA-Z0-9\\.]+$": {
```

---

We need to use `\\` because you have to escape `\` inside json. I think it should be `^ext\\.[a-zA-Z0-9.]+$` (escaping the `.` inside json is redundant).